.github/workflows/release-test.yml

```diff
@@ -184,5 +184,5 @@ jobs:
     - uses: actions/upload-artifact@v4
       if: always()
       with:
-        name: Test Reports
+        name: Test-Reports-${{ matrix.pytest_mark }}
```

Comment:
If spaces are fine, pytest_mark `not iotlab_creds` would probably cause trouble.

Reply:
nvm, you already mention the not-so-pretty situation in the earlier comment. It's GitHub action reports, it doesn't need to be pretty :-). It works, ship it.